presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -2181,6 +2185,92 @@ public void testAggregationWithOrderBy()
                         Optional.empty()));
     }
 
+    @Test
+    public void testCreateRole()
+            throws Exception
+    {
+        assertStatement("CREATE ROLE role", new CreateRole(new Identifier("role"), Optional.empty(), Optional.empty()));
+        assertStatement("CREATE ROLE role1 WITH ADMIN admin",
+                new CreateRole(
+                        new Identifier("role1"),
+                        Optional.of(new GrantorSpecification(
+                                GrantorSpecification.Type.PRINCIPAL,
+                                Optional.of(new PrincipalSpecification(PrincipalSpecification.Type.UNSPECIFIED, new Identifier("admin"))))),
+                        Optional.empty()));
+        assertStatement("CREATE ROLE role2 WITH ADMIN admin1 IN catalog",
```

Comment:
tests are copy-pasted maybe we could extract some method and reuse the code. That way it will be easier to add new test case and read in general.